transaction.go

```diff
@@ -24,6 +24,8 @@ type Transaction struct {
 	Customer                    *Customer                 `xml:"customer,omitempty"`
 	BillingAddress              *Address                  `xml:"billing,omitempty"`
 	ShippingAddress             *Address                  `xml:"shipping,omitempty"`
+	TaxAmount                   *Decimal                  `xml:"tax-amount,omitempty"`
+	TaxExempt                   bool                      `xml:"tax-exempt,omitempty"`
```

Comment:
The `TaxExempt` field should probably be a `*nullable.Bool`, otherwise it won't be sent if it's `false`, and it will be `false` rather than not set if it's not sent by the server.